tests/test_draw.py

```diff
@@ -14,10 +14,10 @@ def get_draw_generator(
     add_noise=True,
     fixed_parameters=False,
     sampling_function=None,
+    rng=np.random.default_rng(0),
```

Comment:
here the seed is set to 0 without justification.

Reply:
my bad, I get it, it's for testing.
Still it may be a good idea to have the test seed apparent somewhere

```python
TEST_SEED = 0
...
rng = np.random.default_rng(TEST_SEED)
```

Reply:
This can be a good idea.